behind-the-docs-1.rst

```diff
@@ -0,0 +1,89 @@
+.. post:: July 1, 2021
+   :tags: behind-the-docs
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Learn more about Read the Docs directly from its team members:
+      appearances in the media during the first half of 2021.
+
+Behind the Docs episode 1
+=========================
+
+During the first half of 2021 several team members of Read the Docs
+have talked about the company in podcasts and online events.
+If you want to learn more about our company and the work we do
+directly from the people that are a core part of it,
+and hear our thoughts on other topics
+like open source sustainability and developer advocacy,
+this blog post is for you.
+
+Eric @ Sustain podcast
+----------------------
+
+`Eric`_ was the special guest of `Episode 66 of Sustain`_ back in February,
+"a podcast about sustaining open source in the long haul".
+Eric started by telling the story of how he and `Anthony`_ co-founded Read the Docs,
+talked about the `Write the Docs <https://www.writethedocs.org/>`_ community,
+described the challenges of launching `EthicalAds <https://www.ethicalads.io/>`_ in 2020,
+and discussed the opportunities of using ethical advertising to fund open source projects.
+
+Listen to Eric online:
+
+.. image:: /img/eric-sustain.png
+   :align: center
+   :target: `Episode 66 of Sustain`_
+
+.. _Episode 66 of Sustain: https://podcast.sustainoss.org/66
+
+"Bootstrapping a sustainable open source project" @ Upstream 2021 online conference
+-----------------------------------------------------------------------------------
+
+Read the Docs and EthicalAds supported `Upstream 2021 <https://upstream.live/>`_ in June,
+a one-day online event about "open source, the developers who use it, and the maintainers who make it".
+`Eric`_ delivered a talk titled `"Bootstrapping a sustainable open source project: The Read the Docs story"`_,
+in which he dived into some of the less known details of how Read the Docs was created,
+shared the different funding strategies the company attempted over the years,
+opened up about the dangers of burnout,
+and detailed the different commercial plans of Read the Docs
+as well as some thoughts about pricing schemes.
+
+Watch Eric online:
+
+.. image:: /img/eric-upstream.png
+   :align: center
+   :target: `"Bootstrapping a sustainable open source project: The Read the Docs story"`_
+
+.. _`"Bootstrapping a sustainable open source project: The Read the Docs story"`: https://explore.tidelift.com/upstream/main/session-eric-holscher
+
+"Making documentation sexy" @ Data for Future podcast
```

Comment:
I don't love this title -- I'd like to keep `sexy` out of anything we're promoting. Definitely fraught and could be seen in a negative light. Probably use `exciting` or something more explicit and descriptive, if possible.

Reply:
Yeah, didn't choose the title of the podcast (different audience). Will just remove it from our heading.